keen/tests/client_tests.py

```diff
@@ -286,6 +286,21 @@ def url_escape(self, url):
             import urllib.parse
             return urllib.parse.quote(url)
 
+@patch("requests.Session.post")
```

Comment:
Oops! We don't need this mock for `EventTests` right now. If you remove it, tests should pass.